content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary_YubiHSM_2_Device.adoc

```diff
@@ -1,49 +1,68 @@
 
-The YubiHSM Setup program is used to perform the initial configuration of the primary YubiHSM 2 device. The YubiHSM 2 is configured so that the necessary key material is generated on the YubiHSM 2 device:
+The YubiHSM Setup program is used to perform the initial configuration of the primary YubiHSM 2 device so that the necessary key material is generated on it:
 
-* One wrap key
-* One application authentication key
-* One audit key
+* One wrap key to be split as described in "Key Splitting and Key Custodians" in https://developers.yubico.com/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc[Prerequisites and Preparations].
+* One application authentication key for authenticating to the YubiHSM 2 through the KSP. This allows the KSP to perform operations in the YubiHSM 2.
 
-The wrap key is split between a predetermined number of key custodians, and each share should be recorded by its custodian. The authentication key and the audit key are exported under wrap to a file located in the current working directory.
+_Note: This initial configuration replaces the default authentication key with a new one, which will only be operable in the same domain as the asymmetric key. The link:../../Concepts/Domain.adoc[domain concept] that is used to compartmentalize the YubiHSM 2 determines this behavior._
+
+TIP: For test purposes you can set the `yubihsm-setup -d` flag to keep the default authentication-key with the administrative privileges; this will allow you to delete keys on the YubiHSM 2 for test purposes only. For production purposes, however, the `yubihsm-setup` command must be executed without the `-d` flag to ensure that the factory preset authentication key is properly deleted from the YubiHSM 2 device._
+
+* One audit key for accessing the internal audit log of the device and resetting the audit log. The audit log retains information about the last 62 operations. It is also used to purge the log if needed. Depending on your local requirements, you may not need to create an audit key. If you are unsure of your requirements, we suggest you create an audit key.
+
+The authentication key and the audit key are exported under wrap to a file in the current working directory on the machine where the YubiHSM Setup program is installed.
+
+TIP: The YubiHSM Setup tool has a help argument that you can call to learn more about its usage.
 
 
 === Overview of Procedure
 
-The configuration steps to be performed after you have inserted the primary factory preset YubiHSM 2 device into the air-gapped system are set out briefly below. These steps are described in detail in the subsequent "Procedure: Configuring Primary YubiHSM 2."
+The configuration steps to be performed  are set out briefly below. These steps are described in detail in the subsequent "Procedure: Configuring Primary YubiHSM 2."
+
+0. Insert the primary factory preset YubiHSM 2 device into the air-gapped system.
```

Comment:
I put in the zero because I need the numbering in the summary overview to match the step numbering in the procedure proper. Now I have corrected it by removing it.